### DISCUSSION TITLE
Is their any solver that helps to simulate Cryogenic Fluid?

### DISCUSSION THREAD

**Discussion by neelshah0034** [pyvista/pyvista#1547] [Q&A] [answered]
Hello everyone,
Does anyone know Is it possible to simulate a spray of cryogenic fluid?
I heard someone has created a flashFoam through OpenFoam, but I am not able to find its repository.

Could anyone please help me out?

Thank you.
Neel Shah

**Comment by akaszynski** [comment, marked as answer]
Seems like this is it: https://zenodo.org/record/4075651

**Reply by neelshah0034** [reply]
Thank you so much for replying.

However, I meant is there any solver available in OpenFoam?

**Reply by tkoyama010** [reply]
If it is about OpenFoam, it is better to ask in [CFD online](https://www.cfd-online.com/Forums/openfoam/140636-basic-openfoam-questions.html).
